lib/S3Plugin.js

```diff
@@ -44,7 +44,10 @@ class S3Plugin {
 
     this.defaultConfig = {
       bucketName: 'your-s3-bucket',
-      region: 'eu-west-3',
+      endpoint: 'https://s3.eu-west-3.amazonaws.com',
+      s3ClientOptions: {
+        's3ForcePathStyle': false
```

Comment:
```suggestion
        s3ForcePathStyle: false
```
It's better to avoid string key style when it's possible